django/forms/fields.py

```diff
@@ -472,19 +473,23 @@ def strptime(self, value, format):
 class DurationField(Field):
     default_error_messages = {
         'invalid': _('Enter a valid duration.'),
+        'overflow': _('days must be between {} and {}'.format(timedelta.min.days, timedelta.max.days))
```

Comment:
@claudep, does this look okay for translators? I'm not sure if it allows apply localization formatting to the numbers.